tests/fake-ssh-add

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/python3
+
```

Comment:
It this added by reuse tool? Would be nice to move it to the commit that added the spdx headers.

Reply:
Yes, added by the reuse tool. In some cases I tried to manually review and fix some headers, I think these I have probably modified it manually in the wrong commit (the previous one), and then `reuse` added the extra line in this commit.